src/graphics/EInkDisplay2.cpp

```diff
@@ -93,6 +93,12 @@ EInkDisplay::EInkDisplay(uint8_t address, int sda, int scl, OLEDDISPLAY_GEOMETRY
     setGeometry(GEOMETRY_RAWMODE, 296, 128);
     LOG_DEBUG("GEOMETRY_RAWMODE, 296, 128\n");
 
+#elif defined(ESP32_S3_PICO)
```

Comment:
The E-Ink display could also be used with the pico, so I suggest not to create a dependency to S3 pico here. Just use a new define for the E-ink display and use it here. This way it is more generic. Also it allows to use S3 pico with other displays.

Reply:
e-ink support has not been added to pico at all yet

Reply:
The S3 pico and pico share the same pinout. So with your extensions (and some platform.ini enhancements) the same E-ink display would also work with the pico I guess.

Reply:
My main point here is not to create artificial dependencies in the code. Keep the display separated from the board.